### PR TITLE
Improve the Kotlin language color + match modern branding

### PR DESCRIPTION
**Pull request by ZacSweers**
The previous color no longer matches the kotlin branding colors and hasn't aged well. This matches the newer branding

Before: `#f18e33`
After: `#A97BFF`

image

## Checklist:

- [x] **I am changing the color associated with a language**
  
  - [x] I have obtained agreement from the wider language community on this color change.
    - I don't know necessarily where to create a wider discussion but happy to post somewhere you recommend!
    - [Added by @lildude]: Discussion started at https://discuss.kotlinlang.org/t/proposed-color-change-for-kotlins-language-on-github/21931
    - https://twitter.com/kotlin

### CODE REVIEW

**Comment by lildude** [comment]
> I don't know necessarily where to create a wider discussion but happy to post somewhere you recommend!

Same thing as your Java colour change PR applies here too; you're more likely to know than us. 😉

**Comment by arturdryomov** [comment]
👀 @elizarov

**Comment by ZacSweers** [comment]
I've started a discussion on the kotlin language discussions forum here: https://discuss.kotlinlang.org/t/proposed-color-change-for-kotlins-language-on-github/21931

I don't know that it has the volume to reach a wider community buy-in, but it does seem the most apt place to seek it at least. 

_For your context, the person @arturdryomov just CC'd is the Project Lead for Kotlin at Jetbrains._

**Comment by lildude** [comment]
Thanks. That should do it. We can see how things go over the next few days.

Copying my responses  from #5384  for completeness:

> We don't have a numerical threshold. We just need to see overwhelming support for the change from the community in that community's chosen preferred location for such discussions. If anyone objects when the change goes live, they will then be able to reference where the discussion took place and continue it or accept it.
>
> We introduced this requirement a few years ago after a colour change was made based on one person's opinion and it went down like a lead balloon in the overall community. We'd like to prevent this from happening again.
>
> Ultimately, this is a community decision, not a GitHub/Linguist decision, so needs to be in that community's forums etc for the greatest exposure.

… and …

> Found it, it was Rust's colour change in #4319 We don't want that to happen again 😁

PS: you can render the css colour inline using backticks, eg`#aa0000`. Would you mind updating the OP to show the two colours so peeps can see the difference in context.

**Comment by ZacSweers** [comment]
Ah neat! Updated both PRs with that

**Comment by LouisCAD** [comment]
Hex color codes are not leading to the color being displayed on the official GitHub mobile app though. I guess it's not the right place to report it, but not sure where it'd be.

**Comment by lildude** [comment]
@LouisCAD  you can try reporting it to https://github.community/ or use the "Report a bug" link in the footer of any GitHub page

**Comment by shipsywor** [comment]
The color that is proposed for the Kotlin in the PR: `#664acc` will end up being too similar to the following language colors.

Kotlin:
color: `#664ACC` (proposed)

Objective-C++:
color: `#6866fb`

Papyrus:
color: `#6600cc`

XQuery:
color: `#5232e7`

ASP.NET:
color: `#9400ff`

Arc:
color: `#aa2afe`

Classic ASP:
color: `#6a40fd`

F#:
color: `#b845fc`

Metal:
color: `#8f14e9`

NumPy:
color: `#9C8AF9`

Though it's true that Kotlin's current language color `#f18e33` could also similarly match other language colors but developers have grown to identify the language by its color over the years. 

It's not that I don't like the color but the incentive is too less and changing color will probably lead to unnecessary confusion. 

I personally do not support this color change.

**Comment by carlonzo** [comment]
Would you mind explaining where this hex is coming from? I tried to search on their website for some branding guidelines but could not find anything significant. is it just from a random pixel from their logo?

**Comment by androiddevnotes** [comment]
> Would you mind explaining where this hex is coming from? I tried to search on their website for some branding guidelines but could not find anything significant. is it just from a random pixel from their logo?

https://www.github.com/github/linguist/tree/2b83aab5dd93cafc0dadb2ab34e6707892a72908/lib%2Flinguist%2Flanguages.yml

**Comment by lildude** [comment]
@shipsywor @carlonzo would you mind continuing this within the community discussion at https://discuss.kotlinlang.org/t/proposed-color-change-for-kotlins-language-on-github/21931 as this really isn't the right place. (I've updated the OP for others to see too).

**Comment by Grigza** [comment]
I suggest to use #A97BFF
We are currently using this color on the site and as an alternative for the Kotlin mark.
I don't think it's a problem that other languages have similar colors. 
Probably whatever color you choose where will be a language with a similar hue.

**Comment by LouisCAD** [comment]
FYI, here's the color suggested by @Grigza displayed for web users to see:`#A97BFF`

File: lib/linguist/languages.yml [hunk 1]
```diff
@@ -2875,7 +2875,7 @@ Kit:
   language_id: 188
 Kotlin:
   type: programming
-  color: "#F18E33"
+  color: "#664ACC"
```
Comment (ZacSweers) [inline lib/linguist/languages.yml:2878]:
```suggestion
  color: "#A97BFF"
```

**Comment by ZacSweers** [comment]
Thanks @Grigza! I've updated the PR with that recommendation

**Comment by lildude** [comment]
> I suggest to use #A97BFF
> We are currently using this color on the site and as an alternative for the Kotlin mark.
> I don't think it's a problem that other languages have similar colors.
> Probably whatever color you choose where will be a language with a similar hue.

I think this, coming from the lead designer at JetBrains, is sufficient to justify accepting this change. 🎉 

Thanks @Grigza, and thanks @ZacSweers for actively engaging to get community buy-in. 🙇

**Review by lildude — APPROVED** [review]

**Comment by itsandreramon** [comment]
Any reasoning for not using the color provided by the buttons on the Kotlin website?  (#8052FF)
Screenshot 2021-06-02 at 12 49 55

**Pull request merged**